packages/front-end/components/Features/FeatureModal/FeatureFromExperimentModal.tsx

```diff
@@ -352,7 +354,10 @@ export default function FeatureFromExperimentModal({
 
       {!existing && (
         <>
-          <FeatureKeyField keyField={form.register("id")} />
+          <FeatureKeyField
+            keyField={form.register("id")}
+            placeHolder={orgSettings.featureKeyExample || "my-feature"}
```

Comment:
It might be better to move this logic into the `FeatureKeyField` component.  

So we would revert changes in this file and in FeatureKeyField, we can grab the org setting and render the placeholder.

Reply:
good idea i will move it.